docs/quickstarts/twitter.rst

```diff
@@ -54,8 +54,51 @@ run:
     $ export OAUTHLIB_INSECURE_TRANSPORT=1
     $ python twitter.py
 
-Visit http://localhost:5000 in your browser, and you should start the OAuth dance
-immediately.
+Visit ``http://127.0.0.1:5000`` in your browser, and you should start the OAuth dance
+immediately after you click on the Signin with Twitter hyperlink.
+
+In your view, make sure you use a plain hyperlink to begin the dance
+so that the redirect loads Twitter's confirmation page in the browser.
+
+.. code-block:: HTML
+
+    <a href="/twitter/login">Sign in with Twitter</a>
+
+You can only use http libraries like axios to check the
+authentication status.
+
+For instance, when the component mounts, you can call a function to check
```

Comment:
Why are we talking about components?

Reply:
Most frontends today involve components (React, Angular, Vuejs are all component based) but yes this should be moved into another doc directed at a modern frontend audience.